### PR TITLE
PD-366   TrueNAS Hardware/Software Compatibility Matrix

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/Hardware/_index.md [hunk 1]
```diff
@@ -3,37 +3,33 @@ title: "TrueNAS Systems"
...
-{{< tab "R-Series" >}}
-<div style="text-align:center;">
+{{< /expand >}}
 
-![R40R50](/images/Hardware/R40R50Front.png "R40 & R50")
+{{< expand "TrueNAS R-Series" "v" >}}
+{{< trueimage src="/images/Hardware/R40R50Front.png" alt="R40 and R50" id="3 - R40 (Front) and R50 (Back)" >}}
 
-</div>
 The TrueNAS R-Series is a family of single controller systems that leverage ZFS and TrueNAS to provide the industry's best storage value.
```
Comment (micjohnson777) [inline content/Hardware/_index.md:33]: possessive case, suggest this as an alternative
```suggestion
The TrueNAS R-Series is a family of single controller systems that leverage ZFS and TrueNAS to provide the best storage value in the industry.
```

**Review by micjohnson777 — DISMISSED** [review]
Looks great, only one thing found with a possessive case. Excellent work!

File: content/Hardware/_index.md [hunk 2]
```diff
@@ -85,29 +75,53 @@ Whether you need a few drives or hundreds, iXsystems offers a variety of customi
...
+| M50 (Gen 2) | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:x"></span> No |
+| M60 (Gen 2) | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:x"></span> No |
+| M30 (Gen 3) | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:check"></span> Yes |
+| M40 (Gen 3) | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:check"></span> Yes |
+| M50 (Gen 3) | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:check"></span> Yes |
+| M60 (Gen 3) | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:check"></span> Yes |
+| X10 | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:x"></span> No |
+| X20 | <span class="iconify" data-icon="oi:check"></span> Yes | <span class="iconify" data-icon="oi:x"></span> No |
+| F1 | <span class="iconify" data-icon="oi:x"></span> No | <span class="iconify" data-icon="oi:check"></span> Yes |
```
Comment (Mrt134) [inline content/Hardware/_index.md:118]:
```suggestion
```

**Review by Mrt134 — CHANGES_REQUESTED** [review]
I want to hold back on this for one final sanity check upstream.

**Pull request closed without merging**